Common/DTO/UserEmailPassword.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DTO
+{
+    class UserEmailPassword
+    {
+        public string Email { get; set; }
+        public string Password { get; set; }
```

Comment:
I think should be only get to not spoil the data

Reply:
I think, you right, because this model used in client